mitodl/mitxpro
# Seed Data updated, data validation added in seed command

#### Pre-Flight checklist


#### What are the relevant tickets?
#2457

#### What's this PR do?
This PR updates the incorrect course keys in the seed json and also adds a validation regex to ensure valid course keys

#### How should this be manually tested?
1. Checkout to this branch,
2. Update seed_data.json with incorrect course keys
3. Run `docker-compose run --rm web ./manage.py seed_data`
4. You should see exceptions for incorrect seed values
